services/movie.py

```diff
@@ -15,13 +18,17 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
-    return Movie.objects.get(id=movie_id)
+    return get_user_model().objects.get(pk=movie_id)
```

Comment:
You are using `get_user_model().objects.get(pk=movie_id)` to get a movie by its id. This is incorrect as you are trying to get a movie from the user model. You should use `Movie.objects.get(pk=movie_id)` instead. This is a bug.